ptx/sec_def_int.ptx

```diff
@@ -45,7 +45,16 @@
   <figure xml:id="fig_defint1">
     <caption>The area under a constant velocity function corresponds to distance traveled</caption>
     <image xml:id="img_defint1" width="47%">
-      <description></description>
+      <shortdescription>
+        Graph of function is linear, the area under the curve is rectangular in shape. 
+      </shortdescription>
+      <description>
+        <p>
+        The y axis is drawn from 0 to 5 represents velocity in feet per second and the x-axis is drawn from 0 to 10 represents time in second.
```

Comment:
You are still missing math tags in the long description. Please use `<m>x</m>`, `<m>y</m>`, etc